versions/3.0.md

```diff
@@ -713,11 +689,12 @@ name: token
 in: header
 description: token to be passed as a header
 required: true
-type: array
-items:
-  type: integer
-  format: int64
-collectionFormat: csv
+schema:
+  type: array
+  items:
+    type: integer
+    format: int64
+  collectionFormat: csv
```

Comment:
with this change, `collectionFormat` may not make sense.  It should be added to the serialization strategy